### PR TITLE
fix: SqlLab Run button fix

### PR DESCRIPTION
**Pull request by AAfghahi**
### SUMMARY
Button width remains consistent even if there is no sql written into SQL lab.  

Disabled effects changed in accordance to the figma. 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

image

image

### ADDITIONAL INFORMATION

- [ x] Has associated issue: https://github.com/apache/superset/issues/12501

### CODE REVIEW

**Review by hughhhh — APPROVED** [review]

**Review by betodealmeida — APPROVED** [review]
Nice!

**Comment by eschutho** [comment]
Looks great! Can we remove the tooltip when it's disabled too?

File: superset-frontend/src/common/components/Dropdown.tsx [hunk 1]
```diff
@@ -82,14 +82,18 @@ const StyledDropdownButton = styled.div`
...
+      }
+      &:nth-child(2) {
         margin: 0;
         border-radius: ${({ theme }) =>
           `0 ${theme.gridUnit}px ${theme.gridUnit}px 0`};
         width: ${({ theme }) => theme.gridUnit * 9}px;
         &:before,
         &:hover:before {
-          border-left: 1px solid ${({ theme }) => theme.colors.grayscale.light5};
+          border-left: 2px solid ${({ theme }) => theme.colors.grayscale.light5};
```
Comment (eschutho) [inline superset-frontend/src/common/components/Dropdown.tsx:96]: Are the mockups 1 or 2px border?

**Comment by AAfghahi** [comment]
The mockups are 2px because that looked closer to the figma.

**Comment by AAfghahi** [comment]
Sorry I just realized mockups === figma.

The mockups has a thicker disabled border.

**Review by eschutho — APPROVED** [review]
lgtm, thank you!

**Pull request merged**